app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -144,7 +149,7 @@
     private AsyncGetEvents event_poll;
 
     private Handler statusUpdateHandler;
-
+    private int toolbarHeightInPx;
```

Comment:
This is nitpicking but I think you forgot to rename this variable starting with 'm' (for member variables).